apps/opitx-md-editor/opitx.yml

```diff
@@ -0,0 +1,13 @@
+name: Opitx Markdown Editor
```

Comment:
Rename the file to `opitx-md-editor.yml`

Reply:
fixed